cmd/server/accounts/validator/strategy.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package validator
+
+import "github.com/moov-io/customers/client"
+
+type StrategyKey struct {
+	Strategy string
+	Vendor   string
+}
+
+type Strategy interface {
+	InitAccountValidation(userID, accountID, customerID string) (*VendorResponse, error)
+	CompleteAccountValidation(userID, customerID string, account *client.Account, accountNumber string, request *VendorRequest) (*VendorResponse, error)
+}
+
+type VendorRequest map[string]interface{}
+type VendorResponse map[string]interface{}
```

Comment:
We could make these interfaces and MarshalJSON will work as expected. The UnmarshalJSON for `VendorResponse` could try each sub-type.